### DISCUSSION TITLE
Heavy help when translating PT-BR

### DISCUSSION THREAD

**Discussion by joaochris** [sircharlo/meeting-media-manager#3225] [Translations]
Another great help when translating PT-BR 😅 

Can you accept **fschein** as a translator?

@sircharlo @mtdvlpr

**Comment by joaochris** [comment]
Can you put it as a proofreader please @mtdvlpr ?

**Comment by mtdvlpr** [comment]
Hi @joaochris, I've made you a language coordinator now in Crowdin. This means you can now also manage the people who translate in PT-BR. You can add and remove people for your language and assign people the proofreader role for PT-BR.

I've done this, because there are a lot of people now translating in PT-BR (there's currently also a new join request). Feel free to use your new permissions to make sure the PT-BR translations remain correct and correctly maintained.

**Reply by joaochris** [reply]
> Hi @joaochris, I've made you a language coordinator now in Crowdin. This means you can now also manage the people who translate in PT-BR. You can add and remove people for your language and assign people the proofreader role for PT-BR.
> 
> I've done this, because there are a lot of people now translating in PT-BR (there's currently also a new join request). Feel free to use your new permissions to make sure the PT-BR translations remain correct and correctly maintained.

Perfect. Thank you very much ;D
